### PR TITLE
Add deploy to Docker button

### LINKED ISSUE #156: Add deploy to docker button
Button for uploading to docker cloud

### PR DESCRIPTION
**Pull request by S2606**
Fixes #156

### CODE REVIEW

**Comment by S2606** [comment]
@gabru-md Please review

**Comment by cclauss** [comment]
Why remove Heroku?  Is it not possible to give people to chose either one?

**Comment by S2606** [comment]
@cclauss Heroku is not removed at all

**Comment by cclauss** [comment]
@S2606 You are correct.  I misread the diff. Thx.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 > The query server can be used to search a keyword/phrase on a search engine (Google, Yahoo, Bing, DuckDuckGo) and get the results as `json` or `xml`. The tool also stores the searched query string in a MongoDB database for analytical purposes. (The search engine scrapper is based on the scraper at [fossasia/searss](https://github.com/fossasia/searss).)
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/query-server)
+[![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/query-server)[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/query-server)
```
Comment (AnshulMalik) [inline README.md]: Can we add a newline between the "code" of a button ? It'd be more readable

Comment (S2606) [reply README.md]: @AnshulMalik The button would be place top-down instead of side by side which will take unnecessary space

**Pull request merged**